docs/upstream.md

```diff
@@ -1,6 +1,6 @@
 # Maintaining your fork's upstream relationship
 
-So you forked this repo. That's awesome. But don't forget you keep your `master` branch in sync with the upstream `master` branch to ensure that your pull requests will always be able to be merged.
+So you forked this repo. That's awesome. But don't forget you keep your `main` branch in sync with the upstream `main` branch to ensure that your pull requests will always be able to be merged.
```

Comment:
This repo still uses a `master` branch. Unless we do the work to rename the branch.